src/features/MentorPage/components/MentorsFilter/Skills/BottomBar/PageSizeDropdown/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { useComponentVisible } from '@/hooks/useComponentShow';
+
+import { pageSizes } from './const';
+
+import { animations, palette } from '@/components/variables';
+import styled from 'styled-components';
+import Text from '@/components/Text';
+import { OpenButton } from './OpenButton';
+import { PageOption } from './PageOption';
+
+type Props = {
+  skillsInPage: number;
+  setSkillsInPage: (nextSize: number) => void;
+};
+const PageSizeDropdown = ({ skillsInPage, setSkillsInPage }: Props) => {
+  const { ref, isComponentVisible, setIsComponentVisible } =
+    useComponentVisible<HTMLDivElement>(false);
+
+  const handlePageSizeChange = (next: number) => {
+    setSkillsInPage(next);
+    setIsComponentVisible(false);
+  };
+
+  return (
+    <Container>
+      <Text>Aihetta sivulla</Text>
+      <Anchor ref={ref}>
+        <OpenButton
+          isComponentVisible={isComponentVisible}
+          onClick={setIsComponentVisible}
+          selected={skillsInPage}
+        />
+
+        {isComponentVisible && (
+          <Menu>
+            {pageSizes.map(size => (
```

Comment:
filter the selected pageSize away from the list of options displayed in the dropdown.
also sort by size

Reply:
Not doing it after all